app/src/routes/routesMap.js

```diff
@@ -104,7 +104,7 @@ const routesMap = {
   [NOT_FOUND]: '/notfound',
 
   ADMINISTRATE_PAGE: redirectRoute('/administrate', () => ({ type: PROJECTS_PAGE })),
-  USER_PROFILE_PAGE: '/user-profile',
+  USER_PROFILE_PAGE: '/user-profile/:profileRoute*',
```

Comment:
As we have predefined routes and don't support any dynamic, let's declare it here.
Refer [SERVER_SETTINGS_TAB_PAGE](https://github.com/reportportal/service-ui/pull/3465/files#diff-256b4742683e2623bba9b071bc6fc31c9fcf42a98e610945b970b50acde3b737R132) with its sub routes.